pkgs/applications/editors/neovim/tests/default.nix

```diff
@@ -222,4 +222,27 @@ rec {
     export HOME=$TMPDIR
     ${nvimWithLuaPackages}/bin/nvim -i NONE --noplugin -es
   '';
+
+  # nixpkgs should install optional packages in the opt folder
+  nvim_with_opt_plug = neovim.override {
```

Comment:
please dont use "plug" as it reminds of "vim-plug" for which I've removed support recently. You can use full name "nvimWithOptionalPlugins" (tests are a mess of snake and camel case, I dont really care either way ^^'')

Reply:
Everything is renamed.